designs/tutorial.stanza

```diff
@@ -6,10 +6,19 @@ defpackage ocdb/designs/tutorial :
   import jitx/commands
   import ocdb/defaults
   import ocdb/generic-components
-  
+  import ocdb/design-vars
+
+MIN-PKG = "0603"
+pcb-landpattern 
```

Comment:
tutorial.stanza should not be modified like this. It's the first design new users see so don't add cruft

Reply:
reverted